CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 CHANGELOG
 =========
+4.0.0 (2022-11-17)
+------------------
+
+* Added support for ip risk database.
```

Comment:
```suggestion
* Added support for the GeoIP2 IP Risk database.
```

Also, it would be good to mention what the breaking change is in particular. Maybe something like:

```
* BREAKING: The methods `ipRisk` and `tryIPRisk` were added to the `DatabaseProvider` interface.
```